modules/test/playwright/tests/export-import-web/main/site.import.spec.ts

```diff
@@ -410,11 +419,259 @@ test('can import a lar file selecting some items to import', async ({
+
+			exportName = `MyExport-${getRandomString()}`;
+
+			await exportImportPage.exportAll(exportName);
+
+			await expect(
+				exportImportPage.page
+					.locator('[data-qa-id="row"]', {
+						hasText: exportName,
+					})
+					.getByText('Successful')
+			).toBeVisible({timeout: 30000});
+
+			exportFilePath =
+				await exportImportPage.downloadExportProcess(exportName);
+		});
+
+		await test.step('Create the site 2 and import the site 1', async () => {
+			site2 = await apiHelpers.headlessSite.createSite({
+				name: getRandomString(),
+			});
+
+			apiHelpers.data.push({id: site2.id, type: 'site'});
+
+			await exportImportPage.goToExport(site2.friendlyUrlPath);
+
+			exportableItems2 = await exportImportPage.getExportableItems();
+
+			await exportImportPage.goToImport(site2.friendlyUrlPath);
+
+			await exportImportPage.import(exportFilePath);
+
+			await expect(
+					exportImportPage.page
+				.locator('[data-qa-id="row"]', {
+					hasText: exportName,
+				})
+				.getByText('Successful')
+			).toBeVisible({timeout: 60000});
+		});
+
+		await test.step('Assert the exportable items from site 1 and site 2 are equal', async () => {
+			await exportImportPage.goToExport(site2.friendlyUrlPath);
+
+			exportableItems3 = await exportImportPage.getExportableItems();
+
+			expect(exportableItems3.size).toEqual(exportableItems1.size);
+
+			for (const [name, count] of exportableItems1.entries()) {
+
+				// TODO LPD-64899
+
+				if (name === 'Calendar') {
+					continue;
+				}
+				else if (name === 'Pages') {
+					expect(exportableItems3.get(name)).toBe(
+						count + exportableItems2.get('Pages')
+					);
+				}
+				else if (name === 'Style Books') {
+
+					// TODO LPD-64905
+
+					expect(exportableItems3.get(name)).toBeGreaterThanOrEqual(
+						count
+					);
+				}
+				else {
+					expect(exportableItems3.get(name)).toBe(count);
+				}
+			}
+		});
+
+		// TODO LPD-65374
+
+		/*
+		await test.step('Assert the home page screenshots from site 1 and site 2 are equal', async () => {
+
+			const comparator = getComparator('image/png');
+
+			const buffer = comparator(
+				await getSiteHomePageScreenshot(page, site1.name, {staging: false}),
+				await getSiteHomePageScreenshot(page, site2.name, {staging: false})
+			);
+
+			if (buffer !== null && buffer.diff !== undefined) {
+				const diffPath = path.join(getTempDir(), `${site1.name}-diff.png`);
+				await fs.writeFile(diffPath, buffer.diff);
+				throw new Error(
+					`The site 1 and site 2 home pages differ. Check the screenshot diff at "${diffPath}".`
+				);
+			}
+		});
+	 	*/
+
+		await test.step('Restore the initial maximum upload request size', async () => {
+			await systemSettingsPage.goToSystemSetting(
+				'Infrastructure',
+				'Upload Servlet Request'
+			);
+
+			await systemSettingsPage.page
+				.getByLabel('Overall Maximum Upload Request Size')
+				.fill(originalOverallMaximumUploadRequestSize);
+
+			await systemSettingsPage.saveButton.click();
+		});
```

Comment:
Could we add a try {} finally {} clause here to make sure that the values are always restored?